xtypes support

lemunozm opened the pull request:
- [ ] https://github.com/osrf/soss/pull/20 merged

lemunozm committed:
fiware xtypes compiling without conversion.

lemunozm committed:
xtypes integration prepared.

lemunozm committed:
Added xtypes conversion.

lemunozm committed:
Fiware working with tests.

lemunozm committed:
Addapt to small API change.

lemunozm committed:
Updated to xtypes standalone soss version.

lemunozm committed:
Updated to work with last xtypes version.

lemunozm committed:
Project layout with packages.

Commit:
Merge pull request #19 from eProsima/feature/packages-layout

Project layout with packages.

lemunozm committed:
Refactor layout.

lemunozm committed:
Updated documentation.

lemunozm committed:
Update diagrams pngs.

Commit:
Merge pull request #20 from eProsima/doc/update

Doc/update

richiware commented:
Build status:
* Linux [![Build Status](http://jenkins.eprosima.com:8080/buildStatus/icon?job=soss-fiware-linux&build=6)](http://jenkins.eprosima.com:8080/job/soss-fiware-linux/6/)

lemunozm committed:
Update sequence diagram.

richiware commented:
Build status:
* Linux [![Build Status](http://jenkins.eprosima.com:8080/buildStatus/icon?job=soss-fiware-linux&build=9)](http://jenkins.eprosima.com:8080/job/soss-fiware-linux/9/)

richiware commented:
Build status:
* Linux [![Build Status](http://jenkins.eprosima.com:8080/buildStatus/icon?job=soss-fiware-linux&build=10)](http://jenkins.eprosima.com:8080/job/soss-fiware-linux/10/)

richiware commented:
Build status:
* Linux [![Build Status](http://jenkins.eprosima.com:8080/buildStatus/icon?job=soss-fiware-linux&build=14)](http://jenkins.eprosima.com:8080/job/soss-fiware-linux/14/)

Commit:
Testing IP to localhost (#24)

* Testing IP to localhost

Jenkins now uses a docker container with contextBroker ready, so the IP for testing is now localhost.

* Localhost to 127.0.0.1

richiware commented:
Build status:
* Linux [![Build Status](http://jenkins.eprosima.com:8080/buildStatus/icon?job=soss-fiware-linux&build=20)](http://jenkins.eprosima.com:8080/job/soss-fiware-linux/20/)

Commit:
Adds Text support. (#25)

richiware commented:
Build status:
* Linux [![Build Status](http://jenkins.eprosima.com:8080/buildStatus/icon?job=soss-fiware-linux&build=22)](http://jenkins.eprosima.com:8080/job/soss-fiware-linux/22/)
